### PR TITLE
Add some docstrings to `keras_core/ops/numpy.py`

### PR DESCRIPTION
**Pull request by guillaumebaquiast**
Add some doctrings to `keras_core/ops/numpy.py`:
- absolute
- abs
- add
- all
- any

I will try to add more doctrings in subsequent PRs once I confirm their style (through this PR).

### CODE REVIEW

File: keras_core/ops/numpy.py [hunk 1]
```diff
@@ -229,6 +229,22 @@ def compute_output_spec(self, x):
 
 @keras_core_export(["keras_core.ops.absolute", "keras_core.ops.numpy.absolute"])
 def absolute(x):
+    """Calculate the absolute value element-wise.
```
Comment (fchollet) [inline keras_core/ops/numpy.py:232]: Prefer using "Compute"

File: keras_core/ops/numpy.py [hunk 2]
```diff
@@ -229,6 +229,22 @@ def compute_output_spec(self, x):
...
+        x: Input tensor.
+
+    Returns:
+        An array containing the absolute value of each element in x.
+
+    Example:
+
+    >>> x = keras_core.ops.convert_to_tensor([-1.2, 1.2])
+    >>> keras_core.ops.absolute(x)
+    Array([1.2, 1.2], dtype=float32)
```
Comment (fchollet) [inline keras_core/ops/numpy.py:246]: Use lowercase `array`

File: keras_core/ops/numpy.py [hunk 3]
```diff
@@ -256,6 +273,31 @@ def compute_output_spec(self, x1, x2):
 
 @keras_core_export(["keras_core.ops.add", "keras_core.ops.numpy.add"])
 def add(x1, x2):
+    """Add arguments element-wise.
+
+    Args:
+        x1, x2: Input tensors.
```
Comment (fchollet) [inline keras_core/ops/numpy.py:279]: One line per argument

File: keras_core/ops/numpy.py [hunk 4]
```diff
@@ -256,6 +273,31 @@ def compute_output_spec(self, x1, x2):
...
+
+    Returns:
+        The tensor containing the element-wise sum of `x1` and `x2`.
+
+    Examples:
+
+    >>> x1 = keras_core.ops.convert_to_tensor([1, 4])
+    >>> x2 = keras_core.ops.convert_to_tensor([5, 6])
+    >>> keras_core.ops.numpy.add(x1, x2)
+    Array([ 6, 10], dtype=int32)
```
Comment (fchollet) [inline keras_core/ops/numpy.py:289]: Use lowercase `array`

File: keras_core/ops/numpy.py [hunk 5]
```diff
@@ -256,6 +273,31 @@ def compute_output_spec(self, x1, x2):
...
+
+    `Add` also broadcasts shapes:
+    >>> x1 = keras_core.ops.convert_to_tensor(
+    ...     [[5,4],
```
Comment (fchollet) [inline keras_core/ops/numpy.py:293]: Use spaces after operators

File: keras_core/ops/numpy.py [hunk 6]
```diff
@@ -290,6 +332,43 @@ def compute_output_spec(self, x):
...
+    Args:
+        x: Input tensor.
+        axis: An integer or tuple of integers that represent the axis along
+            which a logical AND reduction is performed. The default
+            (`axis=None`) is to perform a logical AND over all the dimensions
+            of the input array. `axis` may be negative, in which case it counts
+            for the last to the first axis.
+        keepdims: If True, axes which are reduced are left in the result as
+            dimensions with size one. With this option, the result will
+            broadcast correctly against the input array. Default is False.
```
Comment (fchollet) [inline keras_core/ops/numpy.py:346]: Use backticks for code keywords like `False`

File: keras_core/ops/numpy.py [hunk 7]
```diff
@@ -290,6 +332,43 @@ def compute_output_spec(self, x):
 
 @keras_core_export(["keras_core.ops.all", "keras_core.ops.numpy.all"])
 def all(x, axis=None, keepdims=False):
+    """Test whether all array elements along a given axis evaluate to True.
```
Comment (fchollet) [inline keras_core/ops/numpy.py:335]: Use backticks for code keywords like `True`

File: keras_core/ops/numpy.py (hunk 4 above)
Comment (guillaumebaquiast) [reply keras_core/ops/numpy.py:290]: When using JAX as backend, it prints `Array` with a capital `A`.

Happy to switch to lowercase, but for my understanding, what's the reasoning behind it?

image

Comment (fchollet) [reply keras_core/ops/numpy.py:290]: We want the outputs to be backend-agnostic, e.g. not `tf.Tensor`, etc. `array` is just the generic descriptor we've been using, that's all. It mirrors NumPy.

Comment (guillaumebaquiast) [reply keras_core/ops/numpy.py]: 👍

**Review by fchollet — APPROVED** [review]
Thank you! LGTM.

**Pull request merged**